### PR TITLE
Add a bit of guidance to the auto-merge comment

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

File: src/AutoMerge/util.jl [hunk 1]
```diff
@@ -26,6 +26,16 @@ function _clone_repo_into_dir(url::AbstractString, repo_dir)
     return repo_dir
 end
 
+function _comment_guide()
+    result = string("\n\n",
+                    "Note: If your package works for the current version `x.y.z` of a dependency `foo`, ",
+                    "then a compat entry `foo = x.y.z` implies a compatibility upper bound ",
+                    "for packages following semver. You can additionally include earlier versions",
+                    "your package is compatible with. ",
+                    "See [https://julialang.github.io/Pkg.jl/v1/compatibility/.] for details.")
```
Comment (StefanKarpinski) [inline src/AutoMerge/util.jl]: A multiline `"""` string seems like it would be much easier to read/write here.

Comment (mschauer) [reply src/AutoMerge/util.jl]: This follows the style of the surrounding code it will look a bit odd if we change it here.

Comment (fredrikekre) [reply src/AutoMerge/util.jl]: I believe the reason is that we don't want newlines.

Comment (StefanKarpinski) [reply src/AutoMerge/util.jl]: Ah, ok. I guess that makes sense.

**Pull request merged**